test/check-application

```diff
@@ -1340,6 +1340,11 @@ class TestApplication(testlib.MachineCase):
 
         if self.has_criu:
             b.wait(lambda: self.getContainerAttr("swamped-crate", "State") in NOT_RUNNING)
+            self.toggleExpandedContainer("swamped-crate")
+            b.wait_in_text(
+                f'#containers-containers tr:contains("{IMG_BUSYBOX}") dt:contains("Latest checkpoint") + dd',
+                'today at'
```

Comment:
Can you please also add a check that this description item does *not* appear for an unchecked container? In my local runs I see some rather inconsistent values. For example, I never saw the `State.Checkpointed` value (it may not exist at all for containers without a checkpoint), and the default value of `CheckpointedAt` is "0001-01-01T00:00:00Z", i. e. not quite a null-ish value as podman interprets and formats the zero as time string.

Reply:
Fixed